.github/CONTRIBUTING.rst

```diff
@@ -94,27 +94,28 @@ Documentation
      ^^^^^^^^^^^^^^^^^^^^^
 
      First line of new section.
-- If you add a new feature, demonstrate its awesomeness in the `examples page`_!
+
+- If you add a new feature, demonstrate its awesomeness on the `examples page`_!
 
 
 Changelog
 ^^^^^^^^^
 
-If your change is noteworthy, there needs to be a changelog entry, so our users can learn about it!
+If your change is noteworthy, there needs to be a changelog entry so our users can learn about it!
 
 To avoid merge conflicts, we use the towncrier_ package to manage our changelog.
-``towncrier`` uses independent files for each pull request -- so called *news fragments* -- instead of one monolithic changelog file.
```

Comment:
This one I’ll have to ask to revert for two reasons:

1. if there’s space around the dashes, it’s endash, not emdash.
2. we try to keep RST files ascii (also for apostrophes) because docutils/Sphinx transform them into the proper typographic ones. `--` is endash and I think `---` is emdash.

Reply:
Done! This line also raises the question of "so called" v "so-called", but I don't think the formal hyphenated usage is necessary here (there's no real ambiguity in the so).